content/docs/latest/spire/using/configuring.md

```diff
@@ -380,31 +380,46 @@ A full treatment for Nested SPIRE is beyond the scope of this guide. However thi
 # Export Metrics for Monitoring
 _This configuration applies to the SPIRE Server and SPIRE Agent_
 
-SPIRE allows you to export both Server and  Agent metrics for the purposes of telemetry/monitoring. To configure which metrics collectors you export metrics data to, edit the "Telemetry configuration" section of the server and agent configuration files. SPIRE currently supports exporting of metrics to [Prometheus](https://prometheus.io/), [Statsd](https://github.com/statsd/statsd), and [Datadog](https://docs.datadoghq.com/developers/dogstatsd/) (DogStatsd format). You may configure multiple collectors at the same time. Statsd and DogStatsd both support multiple declarations in the event that you want to send metrics to more than one collector.
+SPIRE allows you to export both Server and Agent metrics for the purposes of telemetry/monitoring. To configure which metrics collectors that SPIRE exports metrics data to, add or edit the telemetry section of the server and agent configuration files. SPIRE can export metrics to 
+[Datadog](https://docs.datadoghq.com/developers/dogstatsd/) (DogStatsd format),
+[M3](https://github.com/m3db/m3),
+[Prometheus](https://prometheus.io/), and
+[Statsd](https://github.com/statsd/statsd).
```

Comment:
You may have already noticed, but according to the READMEs on each repo, the correct spelling is `StatsD` and` DogStatsD` (both ends with capital `D`), but seems like the configs in SPIRE ends with lowercase`d`. I'm not sure what would be the correct way to put it in the docs but we may want to adhere to a single spelling.

Reply:
@sanderson042 Thanks. Can we get capitalization fixed as @mchurichi suggested?

Reply:
Hi AV - sure, done. It's unfortunate that the collector names to use in the `agent.conf` and `server.conf` files are not capitalized correctly (https://github.com/spiffe/spire/blob/master/doc/telemetry_config.md).